src/main/java/org/spongepowered/api/profile/GameProfile.java

```diff
@@ -37,9 +42,28 @@
 public interface GameProfile extends Identifiable, DataSerializable {
 
     /**
-     * Gets the player's last known username.
+     * Creates a {@link GameProfile} from the provided ID and name.
+     *
+     * @param uniqueId The unique ID
+     * @param name The name
+     * @return The created profile
+     */
+    static GameProfile of(UUID uniqueId, String name) {
```

Comment:
I would say yes, as it's what plugins devs would expect after seeing the `Optional` name. There's also a usecase: A plugin might create a `GameProfile`, and then fill it to get any data they didn't know (such as the name).